packages/console/src/components/RadioGroup/Radio.tsx

```diff
@@ -22,12 +22,27 @@ export type Props = {
   children?: ReactNode;
   isChecked?: boolean;
   onClick?: () => void;
+  tabIndex?: number;
 };
 
-const Radio = ({ value, title, name, children, isChecked, onClick }: Props) => {
+const Radio = ({ value, title, name, children, isChecked, onClick, tabIndex }: Props) => {
+  const handleKeyPress: KeyboardEventHandler<HTMLDivElement> = useCallback(
+    ({ key }) => {
+      if ([' ', '\n'].includes(key)) {
```

Comment:
key should be '  ' and 'Enter'? nit: keycode might be more precise.

Reply:
`keyCode` has been [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

Reply:
It doesn't work on my machine.
`Tab` to focus the radio successfully but failed when I press `Enter` to select.

Reply:
fixed, ptal